include/flecs.h

```diff
@@ -3264,6 +3276,8 @@ bool ecs_is_enabled_id(
  * @return The component pointer, NULL if the entity does not have the component.
  *
  * @see ecs_get_mut_id()
+ * @see flecs_get_id_from_record()
```

Comment:
Don't add these to the regular `get` functions. Public API functions should not refer to internal API functions.